.github/workflows/test.yml

```diff
@@ -0,0 +1,134 @@
+
+jobs:
+  check-doc:
+    name: Check doc
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+          override: true
+      - run: cargo install cargo-deadlinks
+      - name: doc (rand)
+        env:
+          RUSTDOCFLAGS: --cfg doc_cfg
+        # --all builds all crates, but with default features for other crates (okay in this case)
+        run: cargo deadlinks --ignore-fragments -- --all --all-features
+
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+            toolchain: stable
+          - os: macos-latest
+            target: x86_64-apple-darwin
+            toolchain: stable
+            # TODO: also aarch64 / M1
+          - os: windows-latest
+            target: x86_64-pc-windows-gnu
+            toolchain: stable
+          - os: windows-latest
+            target: x86_64-pc-windows-msvc
+            toolchain: beta
+            # Test both windows-gnu and windows-msvc; use beta rust on one
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+            toolchain: 1.36.0 # MSRV
+          - os: ubuntu-latest
+            deps: sudo apt-get update ; sudo apt install gcc-multilib
+            target: i686-unknown-linux-gnu
+            toolchain: nightly
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+            toolchain: nightly
+            variant: minimal_versions
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          target: ${{ matrix.target }}
+          toolchain: ${{ matrix.toolchain }}
+          override: true
+      - run: ${{ matrix.deps }}
+      - name: Maybe minimal versions
+        if: ${{ matrix.variant == 'minimal_versions' }}
+        run: cargo generate-lockfile -Z minimal-versions
+      - name: Maybe nightly
+        if: ${{ matrix.toolchain == 'nightly' }}
+        run: |
+          cargo test --target ${{ matrix.target }} --benches
+      - name: Test
+        run: |
+          cargo test --target ${{ matrix.target }} --manifest-path rand_isaac/Cargo.toml --all-features
+          cargo test --target ${{ matrix.target }} --manifest-path rand_xorshift/Cargo.toml --all-features
+          cargo test --target ${{ matrix.target }} --manifest-path rand_xoshiro/Cargo.toml --all-features
+          cargo test --target ${{ matrix.target }} --manifest-path rand_jitter/Cargo.toml --all-features
+
+  test-cross:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: mips-unknown-linux-gnu
+            toolchain: stable
```

Comment:
Since you have only one matrix entry, you can insert these values directly. Well, it may be worth to keep `target` for potential future additions. In that case it will be better to use:
```yaml
matrix:
  target: [mips-unknown-linux-gnu]
```